### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

**Comment by eatonphil** [comment]
Tests passing! Needs a final manual run-through.

File: test/actions/authentication.spec.js [hunk 1]
```diff
@@ -8,15 +8,11 @@ describe('actions/authentication', () => {
   it('should return set token', () => {
     const token = 'token';
     const scopes = 'scopes';
-    const username = 'peanut';
-    const email = 'peanut@domain.com';
```
Comment (na3d) [inline test/actions/authentication.spec.js]: You just didn't like peanut. :p

Comment (eatonphil) [inline test/actions/authentication.spec.js]: I think that is still in a bunch of tests tho

Comment (na3d) [inline test/actions/authentication.spec.js]: hahaha

File: src/layouts/OAuth.js [hunk 2]
```diff
@@ -45,21 +35,26 @@ export class OAuthCallbackPage extends Component {
       return;
     }
 
+    console.log('HERE');
```
Comment (mparke) [inline src/layouts/OAuth.js]: console.log

File: src/layouts/OAuth.js [hunk 3]
```diff
@@ -45,21 +35,26 @@ export class OAuthCallbackPage extends Component {
...
     if (code) {
       const data = new FormData();
       data.append('client_id', clientId);
       data.append('client_secret', clientSecret);
       data.append('code', code);
 
+      console.log('HERE');
```
Comment (mparke) [inline src/layouts/OAuth.js]: console.log

Comment (eatonphil) [inline src/layouts/OAuth.js]: Done and fixed linter to fail on warnings like console logs.

**Comment by mparke** [comment]
Have 4 failing tests still here, is that intended?

**Comment by mparke** [comment]
1 lint error

**Pull request merged**